contribute/documentation.md

```diff
@@ -7,7 +7,6 @@ title: Documentation Contributions
 There are several ways you can help out with the improvement of Scala documentation. These include:
 
 * API Documentation in Scaladoc
-* Code examples and tutorials in activator templates.
```

Comment:
Sbt 13.13 has a `new` command that provides templates. Maybe this is worth mentioning.